db/sql/10_msar_joinable_tables.sql

```diff
@@ -29,7 +29,7 @@ whether to travel from referrer to referant (when False) or from referant to ref
 */
 
 
-CREATE TYPE mathesar_types.joinable_tables AS (
```

Comment:
So that we don't get errors on upgrade, we should drop this type before creating it, right?

Reply:
We never installed this on the user's DBs. Therefore, dropping shouldn't be necessary.

Reply:
Sure, but I'm not talking about upgrading specifically to Mathesar beta. I'm talking about upgrades in general, at any point thereafter. In keeping with the patterns used for `00_msar.sql`, don't we expect `10_msar_joinable_tables.sql` to be run again when the user upgrades to the next version after beta? If not, then what is our expected process for handling changes to the code in that file going forward?

Reply:
You are right, I assumed that the script wouldn't raise an error if the type being installed already existed. I've add the drop statement in 3240918

Reply:
Thanks!